docs/user-manual/quality-test.md

```diff
@@ -0,0 +1,300 @@
+| [Chi2Test](#chi2test) | TH1 | None | ComparisonType, UseUnderflow,<br>UseOverflow |
+
+### PropertyWithinExpectedTest
+This test takes a monitor element and determines one of the following properties:
+
+- Mean
+- Mean90
+- Root mean square (RMS)
+- Root mean square 90 (RMS90)
+- Median
+
+It then compares this property against user-defined criteria. These criteria are one of:
+
+- within a specified range
+- above a threshold value
+- below a threshold value
+
+The result depends on the comparison type. For a range, the result is a p-value of the property being within the required range. For a threshold, the result is 1 if the property passes the threshold, 0 otherwise.
+
+```xml
+<qtest type="PropertyWithinExpectedTest" name="MyMeanTest">
+  <parameter name="Property"> Mean </parameter>
+  <parameter name="ExpectedValue" value="10"/>
+  <parameter name="DeviationLower" value="8"/>
+  <parameter name="DeviationHigher" value="12"/>
+</qtest>
+```
+
+The required parameter `Property` determines which property of the object to calculate. Possible properties are: `Mean`, `Mean90`, `RMS`, `RMS90`, and `Median`. The other three arguments determine the expected value of the property, and the lower and upper bounds, and must be numbers. Which arguments are required depends on the type of test being done. If doing a *within range* test, all three are required. If doing an *above threshold* test, only `DeviationLower` is required. If doing a *below threshold* test, only `DeviationUpper` is required. 
+
+### ExactRefCompareTest
+This takes a monitor element and an attached reference, and compares them to see if they are an exact match. These objects can be either TH1s, TGraphs, or TGraph2Ds, but both the object and it's reference must be of the same type. The result is 1 if the two objects are exactly identical, 0 otherwise.
+
+```xml
+<qtest type="ExactRefCompareTest" name="MyExactRefTest"/>
+```
+
+This test has no parameters. The reference is defined in the `<monitorElement>` section of the XML file, explained [here](#steering-files).
+
+### FitParamInRangeTest
+This takes a monitor element and plots a user-defined function onto it, then gets one of the parameters of the function and checks whether it falls within a user-defined range. 
+
+```xml
+<qtest type="FitParamInRangeTest" name="MyFitParamTest"/>
+  <parameter name ="FitFormula"> gaus(0) </>
+  <parameter name ="GuessParameters" value=""/>
+  <parameter name ="TestParameter" value="1"/>
+  <parameter name ="DeviationLower" value="-0.5"/>
+  <parameter name ="DeviationUpper" value="0.5"/>
+  <parameter name ="FunctionRange" value=""/>
+  <parameter name ="UseLogLikelihood" value="false"/>
+  <parameter name ="UsePearsonChi2" value="false"/>
+  <parameter name ="ImproveFitResult" value="false"/>
+</qtest>
+```
+
+The required parameter `FitFormula` defines the formula used for the fit. This must be [...]. The optional argument `UsePearsonChi2` [...]
```

Comment:
I'm not familiar enough with this qtest to be able to write much about the arguments. Could you contribute something here @rete?